javascript/index.js

```diff
@@ -23,18 +23,117 @@
   getInstruction("mashedPotatoes", 4, (step5) => {
     document.querySelector("#mashedPotatoes").innerHTML += `<li>${step5}</li>`;
     document.querySelector("#mashedPotatoesImg").removeAttribute("hidden");
-  }, (error) => console.log(error));
-
-
+  }, (error) => console.log(error)); */
 
 // Iteration 1 - using callbacks
-// ...
+
+getInstruction("mashedPotatoes", 0, (step1) => {
+  document.querySelector("#mashedPotatoes").innerHTML += `<li>${step1}</li>`;
+  getInstruction("mashedPotatoes", 1, (step2) => {
+    document.querySelector("#mashedPotatoes").innerHTML += `<li>${step2}</li>`;
+    getInstruction("mashedPotatoes", 2, (step3) => {
+      document.querySelector(
+        "#mashedPotatoes"
+      ).innerHTML += `<li>${step3}</li>`;
+      getInstruction("mashedPotatoes", 3, (step4) => {
+        document.querySelector(
+          "#mashedPotatoes"
+        ).innerHTML += `<li>${step4}</li>`;
+        getInstruction("mashedPotatoes", 4, (step5) => {
+          document.querySelector(
+            "#mashedPotatoes"
+          ).innerHTML += `<li>${step5}</li>`;
+          document
+            .querySelector("#mashedPotatoesImg")
+            .removeAttribute("hidden");
+          document.querySelector(
+            "#mashedPotatoes"
+          ).innerHTML += `<li>Mashed Potatoes are ready!</li>`;
+          document
+            .querySelector("#mashedPotatoesImg")
+            .removeAttribute("hidden");
+        });
+      });
+    });
```

Comment:
You are not handling errors here, each function should deal with errors when dealing with promises